CMakeLists.txt

```diff
@@ -227,6 +227,7 @@ if(FLB_ALL)
   set(FLB_IN_NGINX_STATUS 1)
   set(FLB_IN_EXEC         1)
   set(FLB_IN_UNIX_SOCKET  1)
+  set(FLB_IN_SOMEIP       1)
```

Comment:
I don't think you can enable this globally without considering all platforms that are supported including Windows and macOS configuration.